recipes/diligent-core/all/conanfile.py

```diff
@@ -151,37 +155,37 @@ def build(self):
     def package(self):
         cmake = self._configure_cmake()
         cmake.install()
-        tools.rename(src=os.path.join(self.package_folder, "include", "source_subfolder"),
+        rename(self, src=os.path.join(self.package_folder, "include", "source_subfolder"),
         dst=os.path.join(self.package_folder, "include", "DiligentCore"))
 
-        tools.rmdir(os.path.join(self.package_folder, "Licenses"))
-        tools.rmdir(os.path.join(self.package_folder, "lib"))
-        tools.rmdir(os.path.join(self.package_folder, "bin"))
+        rmdir(self, os.path.join(self.package_folder, "Licenses"))
+        rmdir(self, os.path.join(self.package_folder, "lib"))
+        rmdir(self, os.path.join(self.package_folder, "bin"))
         self.copy("License.txt", dst="licenses", src=self._source_subfolder)
 
         if self.options.shared:
             self.copy(pattern="*.dylib", dst="lib", keep_path=False)
             self.copy(pattern="*.so", dst="lib", keep_path=False)
             self.copy(pattern="*.dll", dst="bin", keep_path=False)
-            tools.remove_files_by_mask(os.path.join(self.package_folder, "lib"), "*.a")
+            rm(self, os.path.join(self.package_folder, "lib"), "*.a", recursive=True)
```

Comment:
It's True by default: https://docs.conan.io/en/latest/reference/conanfile/tools/files/basic.html#conan-tools-files-rm 

Of course, be explicit does not harm.